packages/php-wasm/compile/php/php_wasm.c

```diff
@@ -748,10 +781,19 @@ void wasm_set_content_type(char* content_type) {
  * ----------------------------
  *  Sets the request body for the next request.
  *
+ *  This value is expected to be a heap pointer!!!
+ *  If a leftover value exists here without being cleared,
+ *  it will be free'd to prevent memory leaks.
+ *
+ *  If the value is used, it will be free'd in `wasm_destroy_server_context()`.
```

Comment:
I find this unclear. it seems like we're instructing people to free the memory manually, but if they don't, it will be freed automatically.

Which is right? What does someone need to know about allocated memory and the request body? Do they need to do anything? Nothing?